src/forwarder.ts

```diff
@@ -0,0 +1,34 @@
+import Service from "serverless/classes/Service";
+
+const logGroupKey = "AWS::Logs::LogGroup";
+const logGroupSubscriptionKey = "AWS::Logs::SubscriptionFilter";
+
+interface LogGroupResource {
+  Type: typeof logGroupKey;
+  Properties: {
+    LogGroupName: string;
+  };
+}
+
+function isLogGroup(value: any): value is LogGroupResource {
```

Comment:
We probably should narrow the check to `isLambdaLogGroup`, since log groups could be created for other services through serverless.yml? I cannot think of an example right away, but I guess it doesn't hurt to be more strict anyway?

Reply:
I guess the step function log group could be created in this way, since you have to manually create the log group. I'll add a check on the log group name